.circleci/config.yml

```diff
@@ -14,18 +16,23 @@ jobs:
             apk --no-progress update
             apk --no-progress add bash ca-certificates
             nix-channel --update
+      - run:
+          name: Bazel linter
+          command: |
+            nix-shell --pure --run 'bazel run //:buildifier'
       - run:
           name: Build
           command: |
-            nix-shell --pure --run "bazel build --jobs=2 //... @haskell_zlib//... --config=ci"
-            nix-shell --pure --run "bazel build -c dbg --jobs=2 //... --config=ci"
+            nix-shell --pure --run 'bazel build --jobs=2 //... @haskell_zlib//... --config=ci'
+            nix-shell --pure --run 'bazel build -c dbg --jobs=2 //... --config=ci'
       - run:
           name: Run tests
           # bazel does not support recursive bazel call, so we
           # cannot use bazel run here because the test runner uses
           # bazel
           command: |
-            nix-shell --pure --run "bazel build //tests:run-tests && ./bazel-bin/tests/run-tests"
+            # skip running the linter, because it was already done separately, above
```

Comment:
This can be discussed, but it will optimise something like 1s on a really long process. I don't think this optimisation worse the cost of a comment line and a complicated call.

Reply:
You are right, I’ll remove it again.